src/DispersiveShallowWater.jl

```diff
@@ -1,3 +1,15 @@
+"""
+    DispersiveShallowWater
+
+**DispersiveShallowWater.jl** is a Julia package that implements structure-preserving numerical methods for dispersive shallow water models.
+It provides provably conservative, entropy-conserving and well-balanced numerical schemes for some dispersive shallow water models.
+
+The semidiscretizations are based on summation by parts (SBP) operators, which are implemented in SummationByPartsOperators.jl. In order to
```

Comment:
```suggestion
The semidiscretizations are based on summation-by-parts (SBP) operators, which are implemented in SummationByPartsOperators.jl. To
```

Reply:
So would you always write summation-by-parts with hyphens?

Reply:
I would talk about summation by parts or integration by parts as nounds, but write summation-by-parts operators since the SBP stuff is modifying "operators" in this case.